jest.config.js

```diff
@@ -78,17 +86,21 @@ module.exports = {
     './src/components/Table/AsyncTable/AsyncTable.jsx': {
       statements: 56,
       branches: 50,
-      lines: 60,
+      lines: 57,
```

Comment:
is there a reason this coverage exception went down?

Reply:
It's a misnomer due to the formatting. Some code that was previously on one line is now spread across multiple lines and it's catching that the per-line coverage is now lower. There's no actual change in what we're testing.